lib/cmds/update.ts

```diff
@@ -251,3 +269,43 @@ function installAppium(binary: Binary, outputDir: string): void {
       path.resolve(folder, 'package.json'), JSON.stringify({scripts: {appium: 'appium'}}));
   spawn('npm', ['install', 'appium@' + binary.version()], null, {cwd: folder});
 }
+
+interface BinaryPath {
+  last?: string, all?: string[]
+}
+
+interface BrowserFile {
+  chrome?: BinaryPath, standalone?: BinaryPath, gecko?: BinaryPath, iedriver?: BinaryPath
+}
+
+function updateBrowserFile<T extends Binary>(binary: T, outputDir: string) {
+  let downloaded = FileManager.downloadedBinaries(outputDir);
+  let currentDownload = path.resolve(outputDir, binary.executableFilename(Config.osType()));
+  let versions: string[] = [];
+
+  if (!downloaded[binary.id()]) {
+    versions.push(currentDownload);
+  } else if (!downloaded[binary.id()].versions) {
+    versions.push(currentDownload);
+  } else {
+    for (let version of downloaded[binary.id()].versions) {
+      if (version == binary.version()) {
+        versions.push(currentDownload);
+      } else {
+        let defaultExec = binary.executableFilename(Config.osType());
+        let replaceVersion = defaultExec.replace(binary.versionDefault(), version);
+        let defaultFile = path.resolve(outputDir, replaceVersion);
+        versions.push(defaultFile);
+      }
+    }
+  }
+
+  let binaryPath: BinaryPath = {last: currentDownload, all: versions};
+  (browserFile as any)[binary.id()] = binaryPath;
+}
+
+function writeBrowserFile(outputDir: string) {
+  let filePath = path.join(outputDir, 'update-config.json');
+  fs.writeFileSync(filePath, JSON.stringify(browserFile));
+  browserFile = {};
```

Comment:
Done. I hacked this in to pass my tests... removing this and putting in a method to clear out the browserFile. This is still not ideal and will come up with a better fix.